Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/LoadWearablesByParamSystem.cs

```diff
@@ -59,11 +59,22 @@ protected override async UniTask<StreamableLoadingResult<WearablesResponse>> Flo
                    .CreateFromJson<WearableDTO.LambdaResponse>(WRJsonParser.Unity, WRThreadFlags.SwitchToThreadPool);
 
             intention.TotalAmount = lambdaResponse.totalAmount;
+
             for (var i = 0; i < lambdaResponse.elements.Count; i++)
             {
-                WearableDTO wearableDto = lambdaResponse.elements[i].entity;
+                WearableDTO.LambdaResponseElementDto element = lambdaResponse.elements[i];
+                WearableDTO wearableDto = element.entity;
+
                 IWearable wearable = wearableCatalog.GetOrAddWearableByDTO(wearableDto);
 
+                // Add the wearable by shortened and extended urn. This helps the process of retrieving either unique or shared wearables
+                if (element.individualData.Length > 0)
```

Comment:
I could check if `wearable.individualData[0].id != wearable.id` to avoid retrieving the same wearable, but the gain is insignificant, or perhaps im missing something?